README.md

```diff
@@ -1,4 +1,8 @@
-#emailaddress
+# emailaddress
+### This library is Deprecated.
+### You can reach out Digital-Contact team for further assistance alternatively you can use this service.
+### https://github.com/hmrc/preferences-frontend/blob/main/app/emailaddress/EmailAddress.scala
```

Comment:
Do you need this line? Is this an example of using an EmailAddress class without using the library? If so, please add a comment as such.

Reply:
Thanks for the comment, I have been suggested by Paul to add a link instead if someone wants to use it. I will add the comment if someone wants to use the class.

Reply:
Yeah, just a comment to that effect, a bit of context is useful when reading this later. Thanks.